CMakeLists.txt

```diff
@@ -122,7 +122,8 @@ if(USE_AVX)
   endif(USE_LIBXSMM)
 endif(USE_AVX)
 
-if (USE_EPOLL)
+if ((NOT MSVC) AND USE_EPOLL)
+  INCLUDE(CheckIncludeFile)
```

Comment:
what is this?

Reply:
if `INCLUDE(CheckIncludeFile)` is not added, error is `Unknown CMake command 'check_include_file'`.